scripts/gis-material-update/validate-deploy/modules/hsl.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+from modules.config import Config
+from modules.common import *
+from modules.gis_validate_deploy import GisProcessor
+
+class HslBuses(GisProcessor):
+    """Process HSL bus lines."""
+
+    def __init__(self, cfg: Config):
+        self._cfg = cfg
```

Comment:
This class variable isn't used outside `__init__`, so there's no need to initialize it. The `cfg` parameter can be used instead.

Reply:
Changed